airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/CopyConsumer.java

```diff
@@ -161,7 +162,7 @@ public void closeAsOneTransaction(List<StreamCopier> streamCopiers, boolean hasF
         }
       }
       if (!hasFailed) {
-        sqlOperations.executeTransaction(db, mergeCopiersToFinalTableQuery.toString());
+        sqlOperations.executeTransaction(db, Collections.singletonList(mergeCopiersToFinalTableQuery.toString()));
```

Comment:
Can we just use a list of strings instead of wrapping the stringbuilder output in a singleton list?